ios/FluentUI/Vnext/Button/ButtonTokens.swift

```diff
@@ -265,7 +277,43 @@ open class ButtonTokens: ControlTokens {
         }
     }
 
+    /// Defines the shadow used when a floating button is at rest.
     open var restShadow: ShadowInfo { aliasTokens.elevation[.interactiveElevation1Rest] }
 
+    /// Defines the shadow used when a floating button is pressed.
     open var pressedShadow: ShadowInfo { aliasTokens.elevation[.interactiveElevation1Pressed] }
+
+    /// Defiens the minimum height of the button.
+    open var minHeight: CGFloat {
+        switch style {
+        case .primary, .secondary, .ghost:
+            switch size {
+            case .small:
+                return 28
+            case .medium:
+                return 40
+            case .large:
+                return 52
+            }
+        case .accentFloating, .subtleFloating:
+            switch size {
+            case .small, .medium:
+                return 48
+            case .large:
+                return 56
+            }
+        }
+    }
+
+    /// Defines the minimum vertical padding around the content of the button.
+    open var minVerticalPadding: CGFloat {
+        switch size {
+        case .small:
+            return 5
```

Comment:
Where did these values come from? I normally expect new Fluent values to be multiples of 4 whenever possible.

Reply:
They're in the Figma. I can send you a link offline.